Resources/assets/edit-revision.js

```diff
@@ -454,10 +454,9 @@ function addEventListeners(target){
         onFormChange();
     });
 
-    target.find("input").keypress(onFormChange);
-    target.find("input").change(onFormChange);
-    target.find("select").change(onFormChange);
-    target.find("textarea").keypress(onFormChange);
+    target.find("input").on('input', onFormChange);
+    target.find("select").on('change', onFormChange);
+    target.find("textarea").on('input', onFormChange);
```

Comment:
No longer “keypress” and one got removed?

Reply:
input event is much more efficient than the keypress event